filters/json.js

```diff
@@ -0,0 +1,62 @@
+// i18next j4 json format defined at https://www.i18next.com/misc/json-format
+import flatten from "flat";
+
+export class JsonFilter {
+    constructor(params) {
+        this.enableARBAnnotations = params?.enableARBAnnotations || false;
+        this.enablePluralSuffixes = params?.enablePluralSuffixes || false;
+        this.emitComments = params?.emitComments || false;
+    }
+
+    async parseResource({ resource }) {
+        var segments = [];
+        var notes = {};
+        for (const [key, value] of Object.entries(flatten(resource))) {
+            if (key.indexOf("@") === -1) {
```

Comment:
this would treat as an annotation any key that contains `@` and not just at the beginning, per specs.

Reply:
I changed the logic to only treat `@` at the beginning of the key as an ARB attribute